src/app/lib/config/services/persian.js

```diff
@@ -1,6 +1,7 @@
 import { C_POSTBOX } from '@bbc/psammead-styles/colours';
 import { persian as brandSVG } from '@bbc/psammead-assets/svgs';
 import { arabic } from '@bbc/gel-foundations/scripts';
+import * as fonts from '@bbc/psammead-styles/fonts';
```

Comment:
Can we not just do this:
```suggestion
import { F_NASSIM_PERSIAN_REGULAR, F_NASSIM_PERSIAN_BOLD } as fonts from '@bbc/psammead-styles/fonts';
```